### PR TITLE
Hp 3

### PR DESCRIPTION
**Pull request by divyaN73**

### CODE REVIEW

File: scripts/deploy-bnb-chain.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+            maxCap: parseUnits("100,000", 6),
+          },
+        ],
+        equilibriumFee: parseUnits("0.075", 8),
+        maxFee: parseUnits("0.5", 8),
+        transferOverhead: 82491, // TODO
+        maxWalletLiquidityCap: parseUnits("10000", 6), // not used
+        maxLiquidityCap: parseUnits("578829", 6), // trevor
+        svgHelper: await ethers.getContractFactory("BSCUSDT"),
+        decimals: 6,
```
Comment (ankurdubey521) [inline scripts/deploy-bnb-chain.ts:36]: USDT has 18 decimals on BNB Chain

File: scripts/deploy-bnb-chain.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            maxCap: parseUnits("200000", 6),
+          },
+        ],
+        equilibriumFee: ethers.utils.parseUnits("0.075", 8),
+        maxFee: ethers.utils.parseUnits("0.5", 8),
+        transferOverhead: 89491, // TODO
+        maxWalletLiquidityCap: parseUnits("10000", 6), // not used
+        maxLiquidityCap: parseUnits("1643855", 6), // trevor
+        svgHelper: await ethers.getContractFactory("BSCUSDC"),
+        decimals: 6,
```
Comment (ankurdubey521) [inline scripts/deploy-bnb-chain.ts:72]: USDC also has 18 decimals on BNB Chain

**Review by ankurdubey521 — CHANGES_REQUESTED** [review]

File: scripts/deploy-bnb-chain.ts [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+            chainId: 137,
+            minCap: parseUnits("10", 18),
+            // Max Cap needs to be less than the maxTransfer Fee on destination chain id to cover for incentive amount
+            maxCap: parseUnits("50000", 18),
+          },
+          {
+            chainId: 1,
+            minCap: parseUnits("10", 18),
+            // Max Cap needs to be less than the maxTransfer Fee on destination chain id to cover for incentive amount
+            maxCap: parseUnits("100,000", 18),
```
Comment (tomarsachin2271) [inline scripts/deploy-bnb-chain.ts:27]: Remove the  comma from 100,000

File: scripts/deploy-bnb-chain.ts [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        svgHelper: await ethers.getContractFactory("BSCUSDT"),
+        decimals: 18,
+        rewardTokenAddress: "0x06250a4962558F0F3E69FC07F4c67BB9c9eAc739",
+        rewardRatePerSecond: parseUnits("0.004244", 18),
+        excessStateTransferFeePer: parseUnits("0.045", 8),
+      },
+      // USDC
+      {
+        tokenAddress: "0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d",
+        minCap: parseUnits("100", 18),
```
Comment (tomarsachin2271) [inline scripts/deploy-bnb-chain.ts:44]: Make it 10

File: scripts/deploy-bnb-chain.ts [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+          },
+          {
+            chainId: 43114,
+            minCap: parseUnits("10", 18),
+            // Max Cap needs to be less than the maxTransfer Fee on destination chain id to cover for incentive amount
+            maxCap: parseUnits("200000", 18),
+          },
+          {
+            chainId: 1,
+            minCap: parseUnits("10", 18),
```
Comment (tomarsachin2271) [inline scripts/deploy-bnb-chain.ts:61]: Make it 100... if going to Ethereum min amount should be 100 not 10.

File: scripts/deploy-bnb-chain.ts [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+        svgHelper: await ethers.getContractFactory("BSCUSDC"),
+        decimals: 18,
+        rewardTokenAddress: "0x06250a4962558F0F3E69FC07F4c67BB9c9eAc739",
+        rewardRatePerSecond: parseUnits("0.005191", 18), 
+        excessStateTransferFeePer: parseUnits("0.045", 8),
+      },
+      // BICO
+      {
+        tokenAddress: "0x06250a4962558F0F3E69FC07F4c67BB9c9eAc739",
+        minCap: parseUnits("50", 18),
```
Comment (tomarsachin2271) [inline scripts/deploy-bnb-chain.ts:80]: Make it 10.

File: scripts/deploy-bnb-chain.ts [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+            chainId: 137,
+            minCap: parseUnits("10", 18),
+            // Max Cap needs to be less than the maxTransfer Fee on destination chain id to cover for incentive amount
+            maxCap: parseUnits("113220", 18),
+          },
+          {
+            chainId: 1,
+            minCap: parseUnits("10", 18),
+            // Max Cap needs to be less than the maxTransfer Fee on destination chain id to cover for incentive amount
+            maxCap: parseUnits("530,000", 18),
```
Comment (tomarsachin2271) [inline scripts/deploy-bnb-chain.ts:93]: Remove comma from 530,000

File: scripts/deploy-bnb-chain.ts [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+      {
+        tokenAddress: "0x06250a4962558F0F3E69FC07F4c67BB9c9eAc739",
+        minCap: parseUnits("50", 18),
+        maxCap: parseUnits("510000", 18),
+        depositConfigs: [
+          {
+            chainId: 137,
+            minCap: parseUnits("10", 18),
+            // Max Cap needs to be less than the maxTransfer Fee on destination chain id to cover for incentive amount
+            maxCap: parseUnits("113220", 18),
```
Comment (tomarsachin2271) [inline scripts/deploy-bnb-chain.ts:87]: Make it 300000 (300k)

File: scripts/deploy-bnb-chain.ts [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+          },
+          {
+            chainId: 43114,
+            minCap: parseUnits("0.0039", 18),
+            // Max Cap needs to be less than the maxTransfer Fee on destination chain id to cover for incentive amount
+            maxCap: parseUnits("130", 18),
+          },
+          {
+            chainId: 1,
+            minCap: parseUnits("0.0039", 18),
```
Comment (tomarsachin2271) [inline scripts/deploy-bnb-chain.ts:130]: Make it 0.02 when going to ethereum.

**Review by tomarsachin2271 — APPROVED** [review]

**Pull request merged**